database/migrations/0061_historical_data_qty_stats.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations
+
+from database.user_models import Participant
+from libs.file_processing.data_qty_stats import calculate_data_quantity_stats
+
+
+def populate_historical_data_qty_stats(apps, schema_editor):
+    for participant in Participant.objects.all():
+        calculate_data_quantity_stats(participant)
```

Comment:
:lady_beetle: Two things here:
1. You should use `Participant = apps.get_model("Participant")` instead of `from database.user_models import Participant`, otherwise, if the `Participant` model ever changes in the codebase, your migration will break. https://docs.djangoproject.com/en/3.2/topics/migrations/#data-migrations
2. Instead of importing `calculate_data_quantity_stats`, you should copy and paste the function into this migration (for the same reason as above; if the code ever changes, your migration will break). This is one of the few times it's actually appropriate to copy and paste large swaths of code. (Then if you wanted to optimize with a bulk_create in the migration, you can).